pkg/reconciler/revision/resources/deploy.go

```diff
@@ -50,6 +50,28 @@ var (
 		SubPathExpr: "$(K_INTERNAL_POD_NAMESPACE)_$(K_INTERNAL_POD_NAME)_",
 	}
 
+	varProjectedVolume = corev1.Volume{
+		Name: "knative-proj-volume",
+		VolumeSource: corev1.VolumeSource{
+			Projected: &corev1.ProjectedVolumeSource{
+				Sources: []corev1.VolumeProjection{
+					{
+						ServiceAccountToken: &corev1.ServiceAccountTokenProjection{
+							ExpirationSeconds: ptr.Int64(600),
+							Path:              "token",
+							Audience:          "freezer",
+						},
+					},
+				},
```

Comment:
nit: you can collapse the indentation a bit here by doing `[]corev1.VolumeProjection{{` and close with a `}}`. A common idiom used when specifying slices in place.